rust/cubesql/cubesql/src/compile/mod.rs

```diff
@@ -2384,6 +2384,7 @@ mod tests {
                 .dimensions,
             Some(vec![
                 "KibanaSampleDataEcommerce.order_date".to_string(),
```

Comment:
last_mod? Does it mean modified at?

Reply:
Yeah, that was the idea. Think that's good enough or do you propose renaming it?